Branch-SDK/Branch-SDK/BNCStrongMatchHelper.m

```diff
@@ -6,36 +6,95 @@
 //  Copyright © 2015 Branch Metrics. All rights reserved.
 //
 
+
 #import "BNCStrongMatchHelper.h"
+#import <objc/runtime.h>
 #import "BNCConfig.h"
 #import "BNCPreferenceHelper.h"
 #import "BNCSystemObserver.h"
 #import "BranchConstants.h"
 
+
+#pragma mark BNCStrongMatchHelper iOS 8.0
+
+
 // Stub the class for older Xcode versions, methods don't actually do anything.
 #if defined(__IPHONE_OS_VERSION_MAX_ALLOWED) && __IPHONE_OS_VERSION_MAX_ALLOWED < 90000
 
 @implementation BNCStrongMatchHelper
 
-+ (BNCStrongMatchHelper *)strongMatchHelper { return nil; }
-- (void)createStrongMatchWithBranchKey:(NSString *)branchKey { }
-- (BOOL)shouldDelayInstallRequest { return NO; }
-+ (NSURL *)getUrlForCookieBasedMatchingWithBranchKey:(NSString *)branchKey redirectUrl:(NSString *)redirectUrl { return nil; }
++ (BNCStrongMatchHelper *)strongMatchHelper {
+    return nil;
+}
+
+- (void)createStrongMatchWithBranchKey:(NSString *)branchKey {
+}
+
+- (BOOL)shouldDelayInstallRequest {
+    return NO;
+}
+
++ (NSURL *)getUrlForCookieBasedMatchingWithBranchKey:(NSString *)branchKey
+                                         redirectUrl:(NSString *)redirectUrl {
+    return nil;
+}
 
 @end
 
-#else
 
-NSInteger const ABOUT_30_DAYS_TIME_IN_SECONDS = 60 * 60 * 24 * 30;
+#else   // ------------------------------------------------------------------------------ iOS >= 9.0
 
-@interface BNCStrongMatchHelper ()
 
-@property (strong, nonatomic) UIWindow *secondWindow;
+#pragma mark - BNCMatchViewController
+
+
+#import <SafariServices/SafariServices.h>
```

Comment:
I import the header file but don't require that the library be linked to the app.
I tested this on Xcode 7 and Xcode 8, with and without SafariServices linked.